content/docs/learning-guide/samples/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,218 @@
+---
+linkTitle: Kythera City Level Overview
+title: Kythera City Level Overview
+description: Overview of the AI demo level "Kythera City"
+weight: 100
+---
```

Comment:
While we work out some of the formatting and spec for how we want to present official samples on the O3DE site and create the samples index, this should be published as `draft:`. Once we have the samples structured correctly this will switch out of draft and be available on the public site. Until then, `draft` docs are always available to local builds (`hugo serve -D`) and merged docs are in the o3de.org repo.

```suggestion
draft: true
---
```